DSE_6.8_Release_Notes.md

```diff
@@ -26,6 +26,7 @@ If you're developing applications, please refer to the [Java Driver documentatio
 ## 6.8.30 DSE CVE
 * Use com.google.cloud:libraries-bom:26.1.3 libraries for GCE backup and restore functionality. Upgraded Guava from 19.0 to 31.1-jre. (DSP-22904, [CVE-2020-8908](https://nvd.nist.gov/vuln/detail/CVE-2020-8908), [CVE-2018-10237](https://nvd.nist.gov/vuln/detail/CVE-2018-10237), [CVE-2020-7692](https://nvd.nist.gov/vuln/detail/CVE-2020-7692))
 * Updated Apache Ivy of DSE Spark to version 2.5.1. (DSP-22949, [CVE-2022-37865](https://nvd.nist.gov/vuln/detail/CVE-2022-37865), [CVE-2022-37866](https://nvd.nist.gov/vuln/detail/CVE-2022-37866))
+* Removed {{lucene-benchmark}} from {{lucene-solr}} as it contained unnecessary vulnerable library: {{nekohtml:1.9.17}}. (DSP-22902, [CVE-2022-28366](https://nvd.nist.gov/vuln/detail/CVE-2022-28366), [CVE-2022-24839](https://nvd.nist.gov/vuln/detail/CVE-2022-24839), [CVE-2022-29546](https://nvd.nist.gov/vuln/detail/CVE-2022-29546))
```

Comment:
`{{` is not rendered correctly.

Reply:
It needs to be `` in markdown syntax, `{{` is jira's markup language.

Reply:
You are right it does not look good:
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/93650032/207561437-625c74c4-f256-4694-88fa-21265371745b.png">

Reply:
Now it's better:
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/93650032/207562750-c95badc0-270e-4077-8227-ee9aa763411d.png">